acceptance/indy_acceptance/indy-tests/libraries/utils.py

```diff
@@ -0,0 +1,131 @@
+"""
+Created on Nov 9, 2017
+
+@author: khoi.ngo
+
+Containing all functions used by several test steps on test scenarios.
+"""
+from indy.error import IndyError
+from .constant import Colors, Message
+from .result import Status
+
+
+def generate_random_string(prefix="", suffix="", size=20):
+    """
+    Generate random string .
+
+    :param prefix: (optional) Prefix of a string.
+    :param suffix: (optional) Suffix of a string.
+    :param size: (optional) Max length of a string (include prefix and suffix)
+    :return: The random string.
+    """
+    import random
+    import string
+    left_size = size - len(prefix) - len(suffix)
+    random_str = ""
+    if left_size > 0:
+        random_str = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(left_size))
+    else:
+        print("Warning: Length of prefix and suffix more than %s chars" % str(size))
+    result = str(prefix) + random_str + str(suffix)
+    return result
+
+
+def exit_if_exception(code):
```

Comment:
This argument should be named `result` rather than `code`. Actually `result` will contain a code in case it will be of `IndyError` type.

Reply:
Please make it so

Reply:
Updated. Please refer to the latest code.